docs/advanced/inflation.md

```diff
@@ -0,0 +1,421 @@
+# **Inflation**
+
+---
+
+New Decred is created each time a PoW miner discovers a new block - this is commonly referred to as the "block reward". The block reward is split three ways:
+
+* 60% goes to the proof of work miner who found the block
+* 30% goes to the PoS voters on that block (6% to each of the 5 voters)
+* 10% goes towards the development subsidy
+
+The block reward started at 31.19582664 and it adjusts every 6,144 blocks (approximately 21.33 days) by reducing by a factor of 100/101.
+
+If a block includes less than 5 PoS votes there will be a fraction of the block reward which is not allocated to anybody at all. As a result, it is only possible to calculate an upper limit for the total number of Decred which will ever be created. 
+
+> The last block reward will be created in January 2039. The upper limit on the total supply of Decred at that time is 20,999,999.99800912 coins.
+
+The table below shows the block reward and total supply of Decred until the last block reward in 2039. All values in the table are estimates.
+
+Block height | Estimated Date       | Block reward (DCR) | PoW (DCR) | PoS vote (DCR) | Dev subsidy (DCR) | Total DCR supply
+-------------|----------------------|--------------------|-----------|----------------|-------------------|-----------------
+0            | February 8, 2016     |              31.20 |     18.72 |           1.87 |              3.12 |        1,680,000
```

Comment:
Maybe link to a doc (we have one, right?) about the premine here since otherwise it seems odd that there is a total supply at block zero.